README.md

```diff
@@ -1,13 +1,13 @@
-Minetest
-========
+Luanti (formerly Minetest)
+==========================
```

Comment:
wouldn't it suffice to put this into the short description a few lines below?

Reply:
I think it's safer keeping it like this for the moment, to then remove it completely in the future. Basically like we did on Mastodon